### PR TITLE
State in the docs that the value of default Node labels are not reliable.

### PR DESCRIPTION
**Pull request by bsalamat**
ref/ kubernetes/kubernetes#61410

/sig scheduling
cc/ @k82cn @liggitt

### CODE REVIEW

File: docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ with a standard set of labels. As of Kubernetes v1.4 these labels are
 * `beta.kubernetes.io/os`
 * `beta.kubernetes.io/arch`
 
+**Note:** The value of these labels are cloud provider specific and are not guaranteed to be reliable.
```
Comment (zacharysarah) [inline docs/concepts/configuration/assign-pod-node.md]: Please follow the style guide for callouts:
https://kubernetes.io/docs/home/contribute/style-guide/#note

Comment (zacharysarah) [reply docs/concepts/configuration/assign-pod-node.md]: Replace with:
```
**Note:** Labels are specific to cloud providers and are not guaranteed to be reliable.
```

Comment (bsalamat) [reply docs/concepts/configuration/assign-pod-node.md]: I fixed the style, but your suggested text may confuse users thinking the existence of the labels are not guaranteed. The labels with these keys are added, but their values are not guaranteed to be consistent in all environments.

Comment (zacharysarah) [reply docs/concepts/configuration/assign-pod-node.md]: OK--in that case, only one small change:
> The value of these labels is cloud provider specific

**Comment by zacharysarah** [comment]
@bsalamat I'm waiting for a netlify preview to verify that a multi-line note publishes correctly, but it looks like netlify is absent from this PR. I'm working with @chenopis to figure it out.

File: docs/concepts/configuration/assign-pod-node.md (hunk 1 above)
Comment (bsalamat) [reply docs/concepts/configuration/assign-pod-node.md]: Done. Thanks!

**Comment by k82cn** [comment]
LGTM, learned :)

**Pull request merged**